config/environments/production.rb

```diff
@@ -85,6 +97,18 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
+  # Only use :id for inspections in production.
+  config.active_record.attributes_for_inspect = [:id]
```

Comment:
To emulate current behaviour we may need this. which i had applied to assure. Without it I _think_ it may not allow you to not display record details expect for id   - see https://github.com/rails/rails/issues/52728

```suggestion
  config.active_record.attributes_for_inspect = :all
```